contracts/liquidators/registry/LiquidatorsRegistry.sol

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.10;
+
+import "../../midas/SafeOwnableUpgradeable.sol";
+import "../IRedemptionStrategy.sol";
+import { IRouter } from "../../external/solidly/IRouter.sol";
+
+import "openzeppelin-contracts-upgradeable/contracts/token/ERC20/IERC20Upgradeable.sol";
+
+contract LiquidatorsRegistry is SafeOwnableUpgradeable {
```

Comment:
This contract might get really, really big - extensions? Or too early to tell?

Reply:
I thought about it, it is possible.. I will implement the diamond extensions for it